chuck.exs

```diff
@@ -0,0 +1,11 @@
+# Helper script for iex testing
+# $ iex -S mix
+# iex(1)> import_file "chuck.exs"
+
+import Supervisor.Spec
+tree = [worker(Ecto.Ldap.TestRepo, [])]
+opts = [name: Ecto.Ldap.Sup, strategy: :one_for_one]
+Supervisor.start_link(tree, opts)
```

Comment:
I couldn't remember the syntax you were using during our pair programming session, so I opted to borrow some statements I found out on the interwebs that happened to work just fine!